src/components/users/LoginForm.js

```diff
@@ -22,9 +22,9 @@ const LoginForm = () => {
 
   const [isSubmitted, setIsSubmitted] = useState(false);
 
-  const authState = useSelector((state) => state.authentication);
+  const authState = useSelector((state) => state.user);
```

Comment:
```suggestion
  const {.userRequest, errorMsg, requestError } = useSelector((state) => state.user);
```

Reply:
The point after `userRequest` has to be there?

Reply:
No, my bad.